tough/Cargo.toml

```diff
@@ -35,6 +35,7 @@ typed-path = "0.9"
 untrusted = "0.9"
 url = "2"
 walkdir = "2"
+openssl = "0.10"
```

Comment:
So we have in the near past worked to remove openssl from the dependencies of tough and would prefer to keep it that way. Is there a way to adjust this implementation to rely on rustls instead?

Reply:
I believe that reading password encrypted private keys is not yet supported by rustls(pemfile) as per this [issue](https://github.com/rustls/pemfile/issues/15). They've suggested to use pkcs8 instead. 
So, I've done the same and removed openssl.